content/blog/2022/07/2022-07-27-building-naively-android.adoc

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: "Naively Building Android Apps with Jenkins. Not Natively, Naively. - DevOps World 2022"
+tags:
+- devopsworld
+- devopsworld2022
+- android
+- mobile
+authors:
+- gounthar
+opengraph:
+  image: /images/post-images/2022-07-27-naively-building-android/BuildingAndroidAppsNaively.png
+---
+
+NOTE: This is a speaker blogpost for a DevOps World 2022 talk in Orlando, Florida
+
+Come join me at link:https://events.devopsworld.com/widget/cloudbees/devopsworld22/conferenceSessionDetails?tab.day=20220928[DevOps World 2022] for _"Naively Building Android Apps with Jenkins. Not Natively, Naively."_, a talk about my journey with +Jenkins+ while trying to build and publish +Android+ applications without prior +Jenkins+ knowledge.
+
+I've been tinkering with various CI/CD tools for years (+Gitlab CI+, +Circle CI+, +Travis CI+, +Shippable+, +Github Actions+, ...) but not with +Jenkins+ for whatever reason.
```

Comment:
What are these `+` for? Never saw them, and no apparent effect in the preview (?)

Reply:
My bad. It's supposed to be only one `+` to get the mono font...

Reply:
TIL! Which engine supports them?

Reply:
It is supposed to be part of Asciidoc 8.6.9, at least with the GitHub engine... 😊